scripts/webpack/webpack.config.ts

```diff
@@ -118,6 +116,9 @@ const webpackConfig: webpack.Configuration = {
       faker: 'faker/locale/en',
       'react-hook-form': 'react-hook-form/dist/react-hook-form.ie11',
     },
+    fallback: {
+      path: require.resolve('path-browserify'),
```

Comment:
What does this do? Curious if I need it for my changes.

Reply:
I had removed the `node` entry, which would polyfill node apis. I found a few references in the v0 docsite.

Reply:
references to `path`, and the documentation listed this as a workaround.